django_sendfile/__init__.py

```diff
@@ -5,4 +5,4 @@
 del get_versions
 
 # old versions of django-sendfile have this, so keep it for compatibility
-VERSION = tuple(__version__.split("+")[0].split("."))
+VERSION = tuple(__version__.split('+')[0].split('.'))
```

Comment:
Could you remove all these quote changes? They add nothing and will make reviewing a lot harder.

Reply:
Do you want it part of another PR?

Reply:
I don't see what the benefit is. There's no difference between `"string"` and `'string'`.

Reply:
The benefit is code consistency.